src/entity/invoices/invoice.ts

```diff
@@ -160,6 +150,10 @@ export default class Invoice extends PdfAble(BaseEntity) {
   @OneToMany(() => SubTransactionRow, (row) => row.invoice, { cascade: false })
   public subTransactionRows: SubTransactionRow[];
 
+  @ManyToMany(() => SubTransactionRow, { cascade: false })
+  @JoinTable()
+  public subTransactionRowsDeletedInvoice: SubTransactionRow[];
```

Comment:
Maybe it is better to make this a separate entity? Because there are multiple problems with simply using a many-to-many table:
1. You cannot delete the same sub transaction row from the same invoice multiple times (if you are truly dumb and need to do this).
2. You don't know when you deleted a certain sub transaction row, so it is not clear what the invoice was at which point in time.